frameworks/template/src/test/java/com/mesosphere/sdk/template/scheduler/ServiceTest.java

```diff
@@ -1,79 +0,0 @@
-package com.mesosphere.sdk.template.scheduler;
-
-import com.mesosphere.sdk.scheduler.plan.Status;
-import com.mesosphere.sdk.testing.*;
-import org.apache.mesos.Protos;
-import org.junit.Test;
-
-import java.util.ArrayList;
-import java.util.Collection;
-
-public class ServiceTest {
-
-    private static final String VALID_HOSTNAME_CONSTRAINT = "[[\"hostname\", \"UNIQUE\"]]";
-    private static final String INVALID_HOSTNAME_CONSTRAINT = "[[\\\"hostname\\\", \"UNIQUE\"]]";
-
-    @Test
-    public void testSpec() throws Exception {
-        new ServiceTestRunner().run(getDeploymentTicks());
-    }
-
-
-    @Test
-    public void testValidPlacementConstraint() throws Exception {
-        ServiceTestRunner serviceTestRunner = new ServiceTestRunner().setSchedulerEnv("NODE_PLACEMENT", VALID_HOSTNAME_CONSTRAINT);
-        serviceTestRunner.run(getDeploymentTicks());
-    }
-
-    @Test(expected = IllegalStateException.class)
-    public void testInvalidPlacementConstraint() throws Exception {
-        new ServiceTestRunner().setSchedulerEnv("NODE_PLACEMENT", INVALID_HOSTNAME_CONSTRAINT).run(getDeploymentTicks());
-    }
-
-    @Test
-    public void testSwitchToInvalidPlacementConstraint() throws Exception {
-        ServiceTestResult initial = new ServiceTestRunner().setSchedulerEnv("NODE_PLACEMENT", VALID_HOSTNAME_CONSTRAINT).run(getDeploymentTicks());
```

Comment:
One note, I don't think that these are replicated elsewhere -- and as such we should move these to `helloworld` or an alternate location.